Classes/Plugin/Eid/PageViewProxy.php

```diff
@@ -26,6 +31,48 @@
  */
 class PageViewProxy
 {
+    /**
+     * @var RequestFactoryInterface
+     */
+    protected $requestFactory;
+
+    /**
+     * @var mixed
+     */
+    protected $extConf;
+
+    public function __construct()
+    {
+        $this->requestFactory = GeneralUtility::makeInstance(RequestFactory::class);
+        $this->extConf = GeneralUtility::makeInstance(ExtensionConfiguration::class)->get('dlf');
+    }
+
+    protected function overwriteHeaders(array $headers)
+    {
+        header_remove();
+        foreach ($headers as $key => $value) {
+            // The `header()` function should already make sure this is safe
+            // (Won't let $value containing newline pass.)
+            @header("$key: $value");
```

Comment:
```suggestion
            header($key . ': ' . $value);
```
Personally, I don't like variables inside string. Not sure, if this is somewhere written in the coding guidelines. At least, it is not common in TYPO3.

Reply:
Alright, [the TYPO3 guideline is to use single quotes only](https://docs.typo3.org/m/typo3/reference-coreapi/main/en-us/CodingGuidelines/CglPhp/PhpFileFormatting/PhpSyntaxFormatting.html#strings).